sample/F8/speakers

```diff
@@ -0,0 +1,2512 @@
+[{
```

Comment:
Please maintain spacing and indentation i.e. curly braces of an object should be at same indentation.

It is creating problem on loading data from assets when latest data is not downloaded